node/actors/bft/src/testonly/ut_harness.rs

```diff
@@ -274,23 +246,21 @@ impl UTHarness {
         msg: ReplicaCommit,
     ) -> Signed<LeaderCommit> {
         for (i, key) in self.keys.iter().enumerate() {
-            let res = self.consensus.leader.process_replica_commit(
-                ctx,
-                &self.consensus.inner,
-                key.sign_msg(msg),
-            );
-            match (i + 1).cmp(&self.consensus_threshold()) {
+            let res = self.leader.process_replica_commit(ctx, key.sign_msg(msg));
+            let want_threshold = self.replica.inner.threshold();
+            match (i + 1).cmp(&want_threshold) {
                 Ordering::Equal => res.unwrap(),
-                Ordering::Less => assert_matches!(
+                Ordering::Less => res.unwrap(),
+                /*assert_matches!(
```

Comment:
Should this commented out assertion be removed?

Reply:
done